packages/ui-app/src/styles/semantic.css

```diff
@@ -77,3 +79,17 @@
   left: -7px;
   opacity: 1;
 }
+
+.ui--Button-Group {
```

Comment:
Since this is nested under `.ui--` (i.e. our namespace), probably makes more sense (as-is) in `components.css`?